setup.py

```diff
@@ -121,7 +118,7 @@ def run(self):
     author="Red Hat, Inc.",
     author_email="info@ansible.com",
     url="https://github.com/ansible/galaxy_ng/",
-    python_requires=">=3.8",
+    python_requires="<3.9",
```

Comment:
It looks like `python_requires` can use the same format as `requirements` https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires i.e. `python_requires=">=3.8,<3.9"` and I assume we want to require 3.8